cpp/daal/src/algorithms/svm/svm_predict_impl.i

```diff
@@ -66,9 +66,18 @@ public:
         DAAL_CHECK_STATUS_VAR(status);
 
         _shRes->set(kernel_function::values, shResNT);
-        _kernel->getInput()->set(kernel_function::X, xBlockNT);
-        _kernel->getInput()->set(kernel_function::Y, svBlockNT);
-        _kernel->getParameter()->computationMode = kernel_function::matrixMatrix;
+        if (nRows == 1)
```

Comment:
Ok to leave for now, but i would guess that even for 2 rows single threaded version might be better? 
Think this should be part of discussion for other algs on how we would go from single threaded 1 row to multithreaded 100 rows - i.e. where we should put that limit

Reply:
Yes, I would like to leave it for now because current internal svm and kernel APIs are not flexible enough to implement such cases in short time and with appropriate inference performance